scene/resources/packed_scene.cpp

```diff
@@ -416,7 +438,21 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 
 	for (const DeferredNodePathProperties &dnp : deferred_node_paths) {
 		Node *other = dnp.base->get_node_or_null(dnp.path);
-		dnp.base->set(dnp.property, other);
+		const String string_property = dnp.property;
+		if (string_property.contains("/indices/")) {
+			const Vector<String> properties = string_property.split("/");
+			Array array = dnp.base->get(properties[0]);
+
+			if (array.size() >= properties[2].to_int()) {
+				array.push_back(other);
+			} else {
+				array.set(properties[2].to_int(), other);
+			}
```

Comment:
Is it possible for a non-Array property path to include `/indices/` too, and possibly trigger this logic in an unexpected way and crash?

Or can an Array property include `/` in the first place, leading to the `"/indices/`" + itos(k)` suffix being at different indices when splitting by `/`?

Reply:
> Is it possible for a non-Array property path to include /indices/ too, and possibly trigger this logic in an unexpected way and crash?

Yes and no. This code only runs for "deferred NodePath properties", i.e. properties stored as NodePath that are assigned a node after the scene is complete. This limits severely what properties are affected by this code. Users can't use `/` in their properties without using `_get_property_list()`, so it's extremely unlikely that this logic is triggered when it shouldn't. And even if it does, assigning non-Array Variant will result in empty array, I think. So it shouldn't crash.

> Or can an Array property include / in the first place, leading to the "/indices/" + itos(k)suffix being at different indices when splitting by/`?

The `/indicies/` properties are created further above, near your first comment. Extra `/` are only possible if someone uses it wrongly.